soul-common/src/main/java/org/dromara/soul/common/enums/RpcTypeEnum.java

```diff
@@ -37,46 +38,48 @@
     /**
      * Http rpc type enum.
      */
-    HTTP("http", true),
+    HTTP("http", RegisterConstants.HTTP_PATH, true),
```

Comment:
The RPC type here should not be related to the specific path, especially when HTTP can't correspond to a specific path